s3/replication/common/src/systemd/s3_in_memory.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=s3_in_memory server's service file to start a daemon instance.
```

Comment:
systemd service file for in memory s3 server.

Reply:
done